### PR TITLE
Add Dutch Language Support

### PR DESCRIPTION
**Pull request by brandon-qq**
Added Dutch translations to the application, including localization for app content, common phrases, external resources, language settings, and shortcuts.

### CODE REVIEW

**Comment by hyoban** [comment]
Please do not change package.json and remove package-lock.json

File: package-lock.json
Comment (Innei) [inline package-lock.json:1]: No lock file

File: package.json [hunk 1]
```diff
@@ -109,6 +109,7 @@
     "nanoid": "5.0.7",
     "ofetch": "1.3.4",
     "path-to-regexp": "8.1.0",
+    "pnpm": "^9.10.0",
```
Comment (Innei) [inline package.json:112]: no pnpm

File: src/renderer/src/modules/discover/DiscoverFeedForm.tsx [hunk 2]
```diff
@@ -279,10 +290,10 @@ const PreviewUrl: FC<{
   return (
     <div className="group relative min-w-0 pb-4">
       <pre className="w-full whitespace-pre-line break-words text-xs text-theme-foreground/40">
-        {renderedPath}
+        {renderedPath.trim()} {/* Ensure rendered path is trimmed */}
```
Comment (Innei) [inline src/renderer/src/modules/discover/DiscoverFeedForm.tsx:293]: You don't have to do something over and over again.

**Review by Innei — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (brandon-qq) [reply package-lock.json:1]: What do you mean with this?

File: package.json (hunk 1 above)
Comment (brandon-qq) [reply package.json:112]: Is Follow not built on pnpm? Why not use pnpm then

**Comment by Innei** [comment]
@brandon-qq Hi, could you resolve the conflicts? thx

**Comment by brandon-qq** [comment]
image
Literally cannot...? @Innei

**Comment by Innei** [comment]
I am closing this PR due to a long period of time without resolving the conflict. Thank you for your time.

**Pull request closed without merging**